BranchSDK/test/SessionEventTest.cpp

```diff
@@ -0,0 +1,15 @@
+#include <BranchIO/Defines.h>
+#include <BranchIO/Event/SessionEvent.h>
+#include <gtest/gtest.h>
+
+using namespace testing;
+using namespace std;
+using namespace BranchIO;
+
+TEST(SessionOpenEventTest, TestLinkIdentifier)
```

Comment:
Unit test for link_click_id.

Reply:
nice

Reply:
mentioned on JIRA string will be fine for server